wordfreq/_chinese_mapping.py

```diff
@@ -0,0 +1,3275 @@
+SIMPLIFIED_MAP = {13423: '㑔',
```

Comment:
It'll still take about 64 ms to load Jieba, but we might as well not make it take three times that long.